helios-services/src/main/java/com/spotify/helios/rollingupdate/RollingUpdateOpFactory.java

```diff
@@ -160,7 +161,7 @@ public RollingUpdateOp yield() {
 
   private boolean isIgnoreFailures() {
     return deploymentGroup.getRolloutOptions() != null
-           && deploymentGroup.getRolloutOptions().getIgnoreFailures();
+           && TRUE.equals(deploymentGroup.getRolloutOptions().getIgnoreFailures());
```

Comment:
@davidxia this will still throw if getRolloutOptions() returns null, right?

Reply:
see line above